randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
 
 > Dock to Ruined Fountain Access; Heals? False
   * Normal Door to Ruined Fountain Access/Dock to Ruined Fountain
+  > Dock to Arboretum Access
+      Trivial
+  > Dock to Meditation Fountain
+      Trivial
 
 > Dock to Meditation Fountain; Heals? False
   * Normal Door to Meditation Fountain/Dock to Ruined Fountain
+  > Dock to Ruined Fountain Access
+      Trivial
+  > Pickup (Missile Expansion)
+      Morph Ball and Spider Ball and After Flaahgra
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 4; Major Location? False
+  > Dock to Meditation Fountain
+      Morph Ball and Spider Ball
 
 ----------------
 Ruined Shrine
 Asset id: 1014518864
 > Dock to Tower of Light Access; Heals? False
-  * Normal Door to Tower of Light Access/Dock to Ruined Shrine
+  * Wave Door to Tower of Light Access/Dock to Ruined Shrine
+  > Pit
+      Trivial
 
 > Dock to Ruined Shrine Access; Heals? False
   * Normal Door to Ruined Shrine Access/Dock to Ruined Shrine
+  > Dock to Tower of Light Access
+      Morph Ball and Boost Ball and Spider Ball
+  > Pickup (Missile Expansion)
+      Morph Ball and Boost Ball
+  > Pit
+      Trivial
+
+> Pickup (Morph Ball); Heals? False
+  * Pickup 5; Major Location? True
+  > Pit
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 6; Major Location? False
+  > Dock to Ruined Shrine Access
+      Morph Ball
+
+> Pickup (Missile Expansion 2); Heals? False
+  * Pickup 7; Major Location? False
+  > Pit
+      Morph Ball
+
+> Pit; Heals? False
+  > Dock to Ruined Shrine Access
+      Space Jump Boots or Morph Ball
+  > Pickup (Morph Ball)
+      Trivial
+  > Pickup (Missile Expansion 2)
+      Morph Ball Bomb and Morph Ball
 
 ----------------
 Eyon Tunnel
 Asset id: 3407776267
 > Dock to Ruined Nursery; Heals? False
   * Normal Door to Ruined Nursery/Dock to Eyon Tunnel
+  > Dock to Nursery Access
+      Trivial
 
 > Dock to Nursery Access; Heals? False
   * Normal Door to Nursery Access/Dock to Eyon Tunnel
+  > Dock to Ruined Nursery
+      Trivial
 
 ----------------
 Vault
 Asset id: 4010184729
 > Dock to Vault Access; Heals? False
   * Normal Door to Vault Access/Dock to Vault
+  > Dock to Plaza Access
+      Trivial
+  > Pickup (Missile Expansion)
+      Morph Ball Bomb and Morph Ball
 
 > Dock to Plaza Access; Heals? False
   * Normal Door to Plaza Access/Dock to Vault
+  > Dock to Vault Access
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 8; Major Location? False
+  > Dock to Vault Access
+      Trivial
 
 ----------------
 Training Chamber
 Asset id: 1057288964
 > Dock to Training Chamber Access; Heals? False
-  * Normal Door to Training Chamber Access/Dock to Training Chamber
+  * Wave Door to Training Chamber Access/Dock to Training Chamber
+  > Dock to Piston Tunnel
+      After Chozo Ghosts (Training Chamber) and After Unlock Training Chamber morph track
+  > Pickup (Energy Tank)
+      Morph Ball and Spider Ball and After Unlock Training Chamber morph track
```

Comment:
This path should require "after chozo ghosts", not "after morph track". it's independent of leaving the room.

Reply:
wait is it to energy tank or from energy tank?

Reply:
I highlighted the lines to change. this is to the etank

Reply:
Done